scope & closures/ch5.md

```diff
@@ -71,7 +71,7 @@ var baz = foo();
 baz(); // 2 -- Whoa, observamos closure, man.
 ```
 
-The function `bar()` has lexical scope access to the inner scope of `foo()`. But then, we take `bar()`, the function itself, and pass it *as* a value. In this case, we `return` the function object itself that `bar` references.
+A função `bar()` tem acesso léxico ao escopo interno de `foo()`. Mas então, nós temos `bar()`, a função em si, e passamos ela *como* um valor. Nesse caso, nós damos um `return` no objeto referência da função `bar`.
```

Comment:
1. Faltou vírgula antes do `então`.
2. Na parte _`we return the function object itself that bar references`_, pensei em `nós retornamos (return) o próprio objeto da função que bar faz referência`. Pode-se trocar `faz referência` por `referencia` se achar melhor. Pensei nessa tradução para o caso de **_`the function object`_** se referindo à variável `a`, declarada no escopo de `foo`.

Obs.: Achei extremamente difícil a tradução deste parágrafo. Principalmente da primeira frase. A última frase eu também fiquei com dúvida.